tests/ctst/steps/pra.ts

```diff
@@ -328,7 +328,7 @@ Then('the kafka DR volume exists', { timeout: volumeTimeout + 2000 }, async func
     assert(volumeParsed.result!['volume phase'] === 'Bound');
 });
 
-Then('prometheus should scrap federated metrics from DR sink', { timeout: 70000 }, async function (this: Zenko) {
+Then('prometheus should scrap federated metrics from DR sink', { timeout: 180000 }, async function (this: Zenko) {
```

Comment:
Do we understand why 70s is not enough with a 30s frequency? is the system too loaded, are we missing metrics?